p5js/ImageClassification/ImageClassification_VideoScavengerHunt/sketch.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+Webcam Image Classification using MobileNet and p5.js
+This example uses a callback pattern to create the classifier
+=== */
+
+let classifier;
+let video;
+let currentWord;
+let currentIndex = 0;
+let isPlaying = false;
+const words = ['banana', 'watch', 'shoe', 'book', 'cellphone', 'keyboard', 'shirt', 'pants', 'cup'];
+// Create a new p5.speech object
+// You can also control the Language, Rate, Pitch and Volumn of the voice
+// Read more at http://ability.nyu.edu/p5.js-speech/
+const myVoice = new p5.Speech();
+
+function setup() {
+  noCanvas();
+  // Create a camera input
+  video = createCapture(VIDEO);
+  // Initialize the Image Classifier method with MobileNet and the video as the second argument
+  classifier = ml5.imageClassifier('MobileNet', video, modelReady);
+
+  select('#start').mousePressed(() => {
```

Comment:
I'm very torn about this, but at the moment we aren't using arrow syntax for p5.js examples. I think we might want to change this soon though.